recipe/meta.yaml

```diff
@@ -85,7 +85,6 @@ test:
     - test -f $(nf-config --fc)  # [not win]
     - test -f $PREFIX/lib/libnetcdff${SHLIB_EXT}  # [not win]
     - test -f $(nc-config --fc)  # [not win]
-    - test -f $PREFIX/lib/libnetcdf.a  # [not win]
```

Comment:
@zklaus, I assume you're removing this not because it's failing but because you figure it's not necessary?

Reply:
Yes. I even think it's wrong because it is checking for the presence of the static library of a dependency that should come from the `libnetcdf` package, but there has been removed.

Reply:
You're absolutely right. I think it was meant to be libnetcdff.a. I think the check is worth having (if we fix it) since we've had unpleasant surprises in the past.

Reply:
But should we include static libraries?

Reply:
Right. I double checked that the static libs are indeed not included, so I think we are good.